code/modules/mob/living/say.dm

```diff
@@ -359,6 +359,9 @@ proc/get_radio_key_from_channel(var/channel)
 	return src
 
 /mob/living/proc/whisper_say(var/message, var/datum/language/speaking = null, var/alt_name="", var/verb="whispers")
+	if(world.time < say_cooldown)
+		to_chat(src, "<span class='notice'>Take a breath before speaking again!</span>")

```

Comment:
This proc can be called autonomously, please don't put any warnings in it.

Reply:
Addressed